### ISSUE TITLE
[CLOSED] losing energy on misses, dodges and parries on non-finishing moves (like mangle, shred, hemorrhage etc)

### ISSUE THREAD

**Issue by netherspite** [netherspite/cc-buglist#80]
<a href="https://github.com/Isuel"><img src="https://avatars2.githubusercontent.com/u/6793507?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [Isuel](https://github.com/Isuel)**    
_Saturday Jan 17, 2015 at 22:19 GMT_  
_Originally opened as https://github.com/ccshiro/cc-buglist/issues/80_

----

Whenever you miss or the target blocks, dodges or parries your special abilities, you should get back 80% of the energy of the ability, except finishing moves (which can be talented by rogues).

https://www.youtube.com/watch?v=Gqg1vEWEILU

around 3:50, the feral uses mangle on a rogue with evasion at 40 energy, then he has 34 energy left.

at 32:00, notice the rogue dodging the feral's mangle, and the feral getting from 40 energy to 32.

at around 30:00, the rogues are barely losing energy from the druid's dodges.

**Comment by netherspite** [comment]
<a href="https://github.com/Raiilan"><img src="https://avatars3.githubusercontent.com/u/9830264?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Raiilan](https://github.com/Raiilan)**  
_Wednesday Jan 21, 2015 at 22:13 GMT_

----

hmm maybe is something like hitting immune targets like in this old closed issue (maybe)

"shiv energy on immune"

https://github.com/ccshiro/cc-buglist/issues/24

**Comment by netherspite** [comment]
<a href="https://github.com/Isuel"><img src="https://avatars2.githubusercontent.com/u/6793507?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Isuel](https://github.com/Isuel)**  
_Sunday Jan 25, 2015 at 08:56 GMT_

----

no, it happened to me both on players and on dummies and mobs (while i was questing)

**Comment by netherspite** [comment]
<a href="https://github.com/ccshiro"><img src="https://avatars2.githubusercontent.com/u/8135895?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ccshiro](https://github.com/ccshiro)**  
_Sunday Feb 22, 2015 at 19:57 GMT_

----

This is already implemented. To make sure it still works nim and I did the following:
Take 2 chars, one with a lot of dodge (50%), then spam sinister strike. All dodges resulted in only a few points of energy being consumed, whereas if it hit it costed the intended amount.

Basically, seems to be working correctly already.

**Comment by netherspite** [comment]
<a href="https://github.com/Isuel"><img src="https://avatars2.githubusercontent.com/u/6793507?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [Isuel](https://github.com/Isuel)**  
_Tuesday Mar 31, 2015 at 21:15 GMT_

----

kay, after further testing, I noticed that it only happens with mangle(cat): when it gets dodged/parried/misses/fully blocked, you lose all 40 energy of it. tested with rake, and it works fine.

**Comment by netherspite** [comment]
<a href="https://github.com/rylaiii"><img src="https://avatars2.githubusercontent.com/u/10295574?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [rylaiii](https://github.com/rylaiii)**  
_Saturday Sep 24, 2016 at 23:48 GMT_

----

Hello! 

Please check out that place with a  lot of  cool stuff, it's just awesome, take a  look http://outlook.2-stroke-porting.com/lnsevtsi

Take care, ross.porto
